src/main/java/org/candlepin/subscriptions/tally/InventorySwatchDataCollator.java

```diff
@@ -205,6 +206,16 @@ private OrgHostsData trackActiveHypervisor(
     return orgHostsData;
   }
 
+  private int compareStrings(String a, String b) {
+    if (a == null) {
+      return 1;
+    } else if (b == null) {
+      return -1;
+    }
+
+    return a.compareTo(b);
+  }
```

Comment:
Would

```
return Ordering.natural().nullsLast().compare(a, b);
```

work here?  The [Ordering](https://github.com/google/guava/wiki/OrderingExplained) class in Guava has a lot of utility methods that we can use instead of writing our own.

Reply:
For a small thing like this and taking into account that this logic will eventually be removed, I strongly prefer to keep using the basics instead of dealing about what the Ordering, natural, and nullsLast does. 
One example about using third parties libraries is this PR, we're using the PeekingIterator from Guava and had to spend some amount of time to understand why the hasNext was true, but the peek element was empty...

Reply:
Sounds good.